Core/OfficeDevPnP.Core/Framework/Provisioning/ObjectHandlers/ObjectNavigation.cs

```diff
@@ -185,6 +180,48 @@ public override TokenParser ProvisionObjects(Web web, ProvisioningTemplate templ
 
         #region Utility methods
 
+        private bool WebSupportsProvisionNavigation(Web web, ProvisioningTemplate template)
+        {
+            if (template.Navigation == null) return true;
```

Comment:
Ok, got it ... I think I will do a small refactoring here, but nothing really fundamental. Thanks.